frontends/bmd/src/pages/poll_worker_screen.test.tsx

```diff
@@ -580,7 +826,7 @@ test('printing precinct scanner report works as expected with all precinct speci
   });
 
   // Check that the expected results are on the tally report for Precinct 1 Mammal Party
-  const precinct1MammalReports = await screen.getAllByTestId(
+  const precinct1MammalReports = screen.getAllByTestId(
```

Comment:
This was an unrelated opportunistic change to remove some unnecessary `await` expressions (`getAllByTestId` does not return a `Promise`).